src/java.desktop/share/classes/javax/swing/plaf/basic/BasicSeparatorUI.java

```diff
@@ -152,5 +152,13 @@ public Dimension getPreferredSize( JComponent c )
     }
 
     public Dimension getMinimumSize( JComponent c ) { return null; }
-    public Dimension getMaximumSize( JComponent c ) { return null; }
+    public Dimension getMaximumSize( JComponent c ) {
```

Comment:
A blank line before the method would visually separate it from the previous method. The updated implementation isn't trivial.